### PR TITLE
Issue #12: Generate PHP Proto files

### PR DESCRIPTION
**Pull request by frenchfrywpepper**
relates #12

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -48,3 +48,10 @@ pre-publish: test
 .PHONY: thrift
 thrift:
 	thrift -r -gen php -out thrift $(LIGHTSTEP_HOME)/go/src/crouton/crouton.thrift
+
+# An internal LightStep target for regenerating the thrift protocol files
```
Comment (JulianGriggs) [inline Makefile]: Update comment

File: Makefile [hunk 2]
```diff
@@ -48,3 +48,10 @@ pre-publish: test
...
+.PHONY: proto
+proto:
+	protoc --proto_path "$(PWD)/../googleapis:$(PWD)/../lightstep-tracer-common/" \
```
Comment (JulianGriggs) [inline Makefile:59]: Can we make this less dependent on the working directory? If not, can we add a comment indicating what the required directory is?

**Review by JulianGriggs — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (frenchfrywpepper) [reply Makefile]: done

File: Makefile (hunk 2 above)
Comment (frenchfrywpepper) [reply Makefile:59]: done

**Review by JulianGriggs — APPROVED** [review]

**Pull request merged**